### ISSUE TITLE
Opening Second Instance of Visual Studio 2015 Crashes

### ISSUE THREAD

**Issue by yannduran** [madskristensen/SolutionExtensions#10]
With the current nightly build (v1.0.39), if I open a second instance of Visual Studio, once it loads it crashed. The original instance is unaffected.

If I disable the extension, restart VS, then open a second instance, it doesn't crash. If I re-enable the extension, open a second instance of VS, it crashes again.

Repro Steps:
1. Open Visual Studio 2015 RTM with nightly build of v1.0.39 installed
2. Open a second instance of VS 2015 (when it finishes loading, the second instance crashes)
3. Disable extension and restart VS
4. Open second instance of VS 2015 (it loads and no longer crashes)
5. Re-enable the extension and restart VS
6. Open a second instance of VS 2015 (when it finishes loading, the second instance crashes again)

**Comment by yannduran** [comment]
I've updated to v1.0.41 and a second instance of VS still crashes on loading. Disabling the extension once again allows a second instance of VS to be opened.

**Comment by madskristensen** [comment]
Do you open a solution that has a .vsext file?

**Comment by yannduran** [comment]
I don't even get to the chance open a solution, just right-clicking the VS 2015 icon on the task bar to open a second instance of it crashes straight away.

Are you not able to reproduce it?

**Comment by yannduran** [comment]
You got me thinking! In the first instance of Visual Studio, the project that I have open usually **is** in fact a solution that has a _vsext_ file.

**But**, I just tried opening the first instance of VS on its own (ie not opening any project at all), but when I open a second instance it still crashes once it's finished loading. So it doesn't seem to matter if the first instance has a solution with a _vsext_ file in it opened or not. The second instance crashes either way.

I hope that all makes sense.

**Comment by madskristensen** [comment]
Do you have the Beta6 or 7 builds of ASP.NET 5 tooling installed by chance?

**Comment by madskristensen** [comment]
I updated the API referenced .dlls, so can you please try this new build? http://vsixgallery.com/extension/51b81721-cf4e-4ce0-a595-972b1ca2a186/

**Comment by yannduran** [comment]
I installed _v1.0.43_ as you requested. So far I have not had s second instance of VS crash!

Neither of the projects that I'm using have any ASP.NET installed in them. But the last time I saw the ASP.NET tooling it was Beta6 for what it's worth.

But for now, what you've done seems to have solved the situation. I'll let you know if that changes.

No I spoke too soon, opening a third VS (no solution/project selected) still crashes after loading! Damn!

**Comment by yannduran** [comment]
Nah, looks like I spoke too soon. I tried the same steps today, and I'm still getting crashes. :-(

As soon as I disable the extension, the crashes stop.

**Comment by yannduran** [comment]
I just noticed that I didn't mention this before, so I'll do so now, just in case in makes any difference.

I'm running the RTM build of Windows 10 as my current dev environment, in a Hyper-V VM on a Windows 8.1 machine.
